### PR TITLE
Exclude swingui package for coverage analysis

### PR DESCRIPTION
**Pull request by dupliaka**
### Checklist
- [ ] Documentation updated if applicable.
- [ ] Upgrade recipe provided if applicable.

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>specific pull request build</b> please add comment: <b>Jenkins (re)run [optaplanner|apps|examples] tests</b>
* for a <b>full downstream build</b> please add comment: <b>Jenkins run fdb</b>
* for a <b>compile downstream build</b> please add comment: <b>Jenkins run cdb</b>
* for a <b>full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
* for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
* for a <b>Quarkus LTS check</b> please add comment: <b>Jenkins run LTS</b>
* for a <b>specific Quarkus LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner|apps|examples] LTS</b>
* for a <b>Native check</b> please add comment: <b>Jenkins run native</b>
* for a <b>specific Native LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner|apps|examples] native</b>
</details>

### CODE REVIEW

**Comment by dupliaka** [comment]
Jenkins run apps tests

**Comment by dupliaka** [comment]
Jenkins rerun apps tests

**Comment by dupliaka** [comment]
Jenkins retest this

**Comment by dupliaka** [comment]
Jenkins run optaplanner tests

File: .github/workflows/sonarcloud.yml [hunk 1]
```diff
@@ -11,7 +11,8 @@ on:
       - '**.md'
       - '**.adoc'
       - '*.txt'
-      - '.ci/**'      
+      - '.ci/**'
+      - '**/optaplanner-examples/**/swingui/**/*'
```
Comment (rsynek) [inline .github/workflows/sonarcloud.yml:15]: This configuration just defines when the CI for a regular SonarCloud check is skipped. Usually, it runs after a PR has been merged to update the `main` branch analysis (in contrast to the PR check analysis, which checks just the changes introduces in a PR when it's open).

**Review by rsynek — CHANGES_REQUESTED** [review]
Please see e.g. https://docs.sonarqube.org/latest/project-administration/narrowing-the-focus/ for more context.

I think you need to set up a maven property `sonar.coverage.exclusions` in the pom.xml of the `optaplanner-examples`.

File: .github/workflows/sonarcloud.yml (hunk 1 above)
Comment (dupliaka) [reply .github/workflows/sonarcloud.yml:15]: Yes, sorry I got it wrong

**Pull request closed without merging**